### PR TITLE
IBM z hardware clarification

### PR DESCRIPTION
**Pull request by kalexand-rh**
@zzoubkova, @dbenoit17, PTAL?

### CODE REVIEW

**Review by dbenoit17 — APPROVED** [review]
LGTM

**Comment by kalexand-rh** [comment]
@openshift/team-documentation, will you PTAL?

File: modules/installation-requirements-user-infra.adoc [hunk 1]
```diff
@@ -112,7 +112,10 @@ ifdef::ibm-z[]
 [id="minimum-ibm-z-system-requirements_{context}"]
 == Minimum IBM Z system requirements
 
-If you use IBM Z hardware, you can install {product-title} version {product-version} if you use z13 or later. If you use IBM LinuxONE hardware, you can install it on LinuxONE z13 or later.
+You can install {product-title} version {product-version} on the following IBM hardware:
+
+* IBM Z as of z13 and z13s
+* LinuxONE, any version
```
Comment (mburke5678) [inline modules/installation-requirements-user-infra.adoc]: @kalexand-rh 
I assume the technical folks approved these changes? If I read right:
* _IBM z13 or later_ is now z13 and Z13s only? Not z14?
* _LinuxONE z13 or later_ is now z13 earlier or later?

Comment (kalexand-rh) [reply modules/installation-requirements-user-infra.adoc]: @mburke5678, this is the language that IBM requested.

I think it means:

* IBM Z, version z13 and later and z13s and later
* Any version of LinuxONE

Comment (kalexand-rh) [reply modules/installation-requirements-user-infra.adoc]: @dbenoit17, do you object to changing  "IBM Z as of z13 and z13s" to "IBM Z, version z13 and later and z13s and later"

Comment (kalexand-rh) [reply modules/installation-requirements-user-infra.adoc]: David +1'd this change via DM.

**Comment by kalexand-rh** [comment]
This change will need to be present in 4.3 when it is released.

**Pull request merged**